exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -185,7 +185,8 @@ private Metadata(FileSystem fs, ParquetFormatConfig formatConfig) {
         childFiles.add(file);
       }
     }
-    ParquetTableMetadata_v3 parquetTableMetadata = new ParquetTableMetadata_v3(true);
+    ParquetTableMetadata_v3 parquetTableMetadata = new ParquetTableMetadata_v3(DrillVersionInfo.getVersion(),
+        ParquetWriter.WRITER_VERSION);
```

Comment:
`is.date.correct` or `parquet-writer.version` were needed in metadata cache file for quick detection of date values correctness. Otherwise need to check `files.rowGroups.columns.mxValue` values from this cache file. 
But thought a little, I've understood that due to new added `ParquetTableMetadata_v3` we can check:
If version of parquet metadata cache file is 3, the date values are definitely correct. Otherwise (when parquet metadata cache file was generated earlier) need to check date values from this file. 
So `writerVersion` is redundant in the `ParquetTableMetadataBase` now. I deleted it. Please approve does it make sense?